README.rst

```diff
@@ -18,15 +18,36 @@ Installation
 Usage
 =====
 
-::
+Running a match:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> p1 = axlf.Player('k31r')
+   >>> p2 = axlf.Player('k33r')
+   >>> match = axl.Match((p1, p2), turns=5)
+   >>> match.play()
+   [(C, C), (C, C), (C, D), (C, D), (C, C)]
+
+Reproducing Axelrod's second tournament:
+
+.. code-block:: python
+
+   import axelrod as axl
```

Comment:
Can we write all code with python `>>>` so that we can run doctests to catch errors.